### DISCUSSION TITLE
2022/04/02/%E7%94%A8rust%E6%90%9E%E4%B8%AA%E6%95%B0%E6%8D%AE%E7%BB%93%E6%9E%84-%E9%93%BE%E8%A1%A83/

### DISCUSSION THREAD

**Discussion by giscus** [OOJ2003/OOJ2003.github.io#17] [Announcements]
# 2022/04/02/%E7%94%A8rust%E6%90%9E%E4%B8%AA%E6%95%B0%E6%8D%AE%E7%BB%93%E6%9E%84-%E9%93%BE%E8%A1%A83/

为链表实现迭代器相关 trait

https://ooj2003.github.io/2022/04/02/%E7%94%A8rust%E6%90%9E%E4%B8%AA%E6%95%B0%E6%8D%AE%E7%BB%93%E6%9E%84-%E9%93%BE%E8%A1%A83/

**Comment by OOJ2003** [comment]
才发现这个竟然没继续写了。。。之后有时间补上

**Reply by OOJ2003** [reply]
下个月一定补上(
